### DISCUSSION TITLE
Regarding Application For Github Global Campus Process

### DISCUSSION THREAD

**Discussion by Ratikant-rout** [community/community#72619] [GitHub Education]
### Hi everyone,

The Continue Button Is Not Working So What to do Next?
![2023-10-26 (4)](https://github.com/community/community/assets/111556535/ccea9080-d982-4197-87c7-a07190c7f90b)
![2023-10-26 (3)](https://github.com/community/community/assets/111556535/43cca9b7-0a50-4a2e-be54-ef8a0f907ba4)

![2023-10-26 (2)](https://github.com/community/community/assets/111556535/385dc1be-8458-4db3-8584-c1a41cefbdcc)

SO WHAT TO DO NEXT AND AFTER HOW MANY DAYS THE CONTINUE BUTTON WILL WORK?

**Comment by Kapileswar-Moharana** [comment]
## It appears that you're encountering an issue with a "Continue" button not working, but your message is somewhat unclear, and there isn't enough context to provide a specific solution. Here are some general steps you can follow to troubleshoot issues with non-functional buttons on a website or application
- Refresh the Page:
- Clear Browser Cache:
- Check for JavaScript Errors: 
- Try a Different Browser:
- Check for Browser Extensions:
- Test on a Different Device:
- Contact Support:
- Wait and Try Later: If you're asking about a specific time frame for when the "Continue" button will work, that depends on the nature of the issue. 
## For a more specific solution, it's essential to provide more details about the website or application you're using, the specific problem you're facing, and any error messages or unusual behavior you're encountering.

**Reply by Ratikant-rout** [reply]
I Think It Was Freezed For 24 Hours Let's See What Happens After 24 Hrs Then I Will Message You

**Reply by Ratikant-rout** [reply]
![2023-10-26 (7)](https://github.com/community/community/assets/111556535/4337015d-b8c5-42c0-9055-9bccbc84102e)

**Reply by Kapileswar-Moharana** [reply]
Ok Then...

**Reply by queenofcorgis** [reply]
@Kapileswar-Moharana

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.
